stories/.storybook/webpack.config.js

```diff
@@ -37,7 +37,7 @@ module.exports = ({config}) => {
           options: {
             ignoreDiagnostics: ['6133'],
             compilerOptions: {
-              rootDir: '../../giraffe',
+              rootDir: '..',
```

Comment:
What's the difference between this and just deleting the whole line?

Reply:
There is a separate PR already merged for this https://github.com/influxdata/giraffe/pull/500